packages/web-app-files/src/components/Search/List.vue

```diff
@@ -24,9 +24,14 @@
             <oc-avatar :width="24" :user-name="item.label" />
           </template>
           <template #item="{ item }">
-            <div v-text="item.label" />
+            <span v-text="item.label" />
           </template>
         </item-filter>
+        <item-filter-toggle
+          :filter-label="$gettext('Search in file content')"
+          filter-name="fulltext"
+          class="files-search-filter-fulltext oc-mr-s"
+        />
```

Comment:
Tika seems to required for full-text search, I am wondering if we shouldn't rely on a capability then to render this toggle only if capability is enabned

Reply:
I've created https://github.com/owncloud/ocis/issues/6367, Julian will tackle it as soon as he finds the time.

Reply:
guys! love it 😍